templates/web/pages/tag/tag.tt.html

```diff
@@ -48,6 +50,43 @@
             [% description %]
         </div>
     [% END %]
+    [% IF tagid.defined  %]
+        <!-- injecting facet-knowledge-panel -->
+        <div id="facet-knowledge-panel">
+            <h2 id="facet_panels_title"></h2>
+            <div id="facet_panels_content"></div>
+        </div>
+        <!-- Fetching facet knowledge panel -->
+        <script>
+            let facet_kp_url = "https://facets-kp.openfoodfacts.net/render-to-html?";
+            let params = "facet_tag=[% tagtype %]&value_tag=[% canon_tagid %]";
+            if ("[% tagid2.defined %]"){
+                params += "&sec_facet_tag=[% tagtype2 %]&sec_value_tag=[% canon_tagid2 %]"
+            }
```

Comment:
@stephanegigandet @alexgarel `[% canon_tagid2 %]` is not giving any value.

Reply:
I commented here: https://github.com/openfoodfacts/openfoodfacts-server/pull/8055/files#r1097751904